src/basic/terminal-util.h

```diff
@@ -35,13 +35,16 @@
 #define ANSI_HIGHLIGHT_BLACK             "\x1B[0;1;30m"
 #define ANSI_HIGHLIGHT_RED               "\x1B[0;1;31m"
 #define ANSI_HIGHLIGHT_GREEN             "\x1B[0;1;32m"
-#define ANSI_HIGHLIGHT_YELLOW            "\x1B[0;1;38;5;185m"
+#define ANSI_HIGHLIGHT_YELLOW            "\x1B[0;1;33m"
 #define ANSI_HIGHLIGHT_BLUE              "\x1B[0;1;34m"
 #define ANSI_HIGHLIGHT_MAGENTA           "\x1B[0;1;35m"
 #define ANSI_HIGHLIGHT_CYAN              "\x1B[0;1;36m"
 #define ANSI_HIGHLIGHT_WHITE             "\x1B[0;1;37m"
-#define ANSI_HIGHLIGHT_GREY              "\x1B[0;1;38;5;245m"
 #define ANSI_HIGHLIGHT_YELLOW4           "\x1B[0;1;38;5;100m"
+#define ANSI_HIGHLIGHT_KHAKI3            "\x1B[0;1;38;5;185m"
+#define ANSI_HIGHLIGHT_GREY              "\x1B[0;1;38;5;245m"
```

Comment:
note that this is a replacement too, i.e. if all was good we could use "1;30m" for this... i.e. "bright black"